apps/ledger-live-mobile/src/screens/Analytics/Operations.tsx

```diff
@@ -25,7 +25,7 @@ import NoMoreOperationFooter from "../../components/NoMoreOperationFooter";
 
 import EmptyStatePortfolio from "../Portfolio/EmptyStatePortfolio";
 import NoOpStatePortfolio from "../Portfolio/NoOpStatePortfolio";
-import OperationRow from "../../components/OperationRow";
+import OperationRow from "../../components/OperationRow/OperationRow";
```

Comment:
A little weird to have `OperationRow/OperationRow.tsx` maybe use `OperationRow/index.tsx` instead ?

Reply:
Yes I agree it was weird to me at first
I made the choice because it's easier to do a search if we have unique filename vs hundreds of index.tsx. But I agree index.tsx is more standard so we can go with either one afaik

What do you think ?

```
OperationRow
     OperationRow.tsx
     OperationRow.test.tsx

OperationFooBar
     OperationFooBar.tsx
     OperationFooBar.test.tsx
```

vs

```
OperationRow
     index.tsx
     OperationRow.test.tsx

OperationFooBar
     index.tsx
     OperationFooBar.test.tsx
```